frontend/tutor-milton/src/components/googleLogin.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react"
+import firebase from "../config/firebase-config"
+import StyledFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+
+
+class googleLogin extends Component {
+  state = { isSignedIn: false }
+  uiConfig = {
+    signInFlow: "popup",
+    signInOptions: [
+      firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+    ],
+    callbacks: {
+      signInSuccess: () => false
+    }
+  }
+
+  componentDidMount = () => {
+    firebase.auth().onAuthStateChanged(user => {
+      this.setState({ isSignedIn: !!user })
+      console.log("user", user)
+    })
+  }
+
+  render() {
+    return (
+      <div className="container1">
+        {this.state.isSignedIn ? (
+          <span>
+            <div>Signed In!</div>
+            <button onClick={() => firebase.auth().signOut()}>Sign out!</button>
+            <h1>Welcome {firebase.auth().currentUser.displayName}</h1>
+            <img
+              alt="profile picture"
+              src={firebase.auth().currentUser.photoURL}
+            />
+          </span>
+        ) : (
+          <div className="container">
+          <StyledFirebaseAuth className="sign-Button"
```

Comment:
for class names I think we should stick to the convention of lowercase-lowercase so in this case "sign-in-button" instead of capitalizing "Button."